app/models/feed.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+class Feed < ActiveRecord::Base
```

Comment:
I'm not entirely convinced of how useful this method is for other feeds that may not have human readable versions.
Or where the human readable version can't be easily derived from the RSS URL. At some point it may make sense to add the html_url for each feed to the DB.
In any case, for the moment this is indeed quite useful for our purposes and the feeds we're currently dealing with, so I'll add this.